frontend/app/utils/env.server.ts

```diff
@@ -71,6 +71,8 @@ const serverEnv = z.object({
     .string()
     .transform((val) => val.split(',').map((val) => val.trim()))
     .default('http://localhost:3000/auth/callback/raoidc'),
+
+  SCCH_BASE_URI: z.string().url().default('https://www.example.com'),
```

Comment:
Here you have no trailing slash, but in `.env.example` you do have a trailing slash.